books/free-programming-books-langs.md

```diff
@@ -421,6 +421,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 ### C
 
 * [256-Color VGA Programming in C](http://www.brackeen.com/vga/) - David Brackeen
+* [3D Math Primer for Graphics and Game Development](https://gamemath.com/book/intro.html) - Fletcher Dunn (HTML)
```

Comment:
Is it in the most appropriate section? "Game Development" category in `free-programming-books-subjects` is better?

Reply:
I'd go with game dev

Reply:
Done